ha/setup/create_pacemaker_resources.py

```diff
@@ -0,0 +1,251 @@
+    SimpleCommand().run_cmd(cmd_fsm)
+
+    constraints = [
+            f"pcs -f {cib_xml} constraint order motr-ios then motr-free-space-mon",
+            f"pcs -f {cib_xml} constraint colocation add motr-free-space-mon with motr-ios"
+            ]
+    for c in constraints:
+        SimpleCommand().run_cmd(c)
+
+    if push:
+        cib_push(cib_xml)
+
+
+def s3servers(cib_xml, push=False):
+    """Create resources that belong to s3server group and clone the group.
+
+    S3 background consumer is ordered after s3server and co-located with it."""
+    for i in range(1, 12):
+        cmd_s3server = f"pcs -f {cib_xml} resource create s3server-{i} ocf:seagate:dynamic_fid_service_ra service=s3server fid_service_name=s3service --group s3server"
+        SimpleCommand().run_cmd(cmd_s3server)
+
+    cmd_s3bc = f"pcs -f {cib_xml} resource create s3backcons systemd:s3backgroundconsumer meta failure-timeout=300s --group s3server"
+    SimpleCommand().run_cmd(cmd_s3bc)
+
+    cmd_clone_s3server = f"pcs -f {cib_xml} resource clone s3server"
+    SimpleCommand().run_cmd(cmd_clone_s3server)
+
+    if push:
+        cib_push(cib_xml)
+
+
+def s3bp(cib_xml, push=False):
+    """Create S3 background producer
+
+    S3 background producer have to be only 1 per cluster and co-located with
+    s3server."""
+    cmd_s3bp = f"pcs -f {cib_xml} resource create s3backprod systemd:s3backgroundproducer op monitor interval=30s"
+    cmd_order = f"pcs -f {cib_xml} constraint order s3server-clone then s3backprod"
+
+    for s in (cmd_s3bp, cmd_order):
+        SimpleCommand().run_cmd(s)
+
+    if push:
+        cib_push(cib_xml)
+
+
+def s3auth(cib_xml, push=False):
+    """Create haproxy S3 auth server resource in pacemaker."""
+    cmd_s3auth = f"pcs -f {cib_xml} resource create s3auth systemd:s3authserver clone op monitor interval=30"
+    SimpleCommand().run_cmd(cmd_s3auth)
+
+    if push:
+        cib_push(cib_xml)
+
+
+def haproxy(cib_xml, push=False):
+    """Create haproxy clone resource in pacemaker."""
+    cmd_haproxy = f"pcs -f {cib_xml} resource create haproxy systemd:haproxy clone op monitor interval=30"
+    SimpleCommand().run_cmd(cmd_haproxy)
+
+    if push:
+        cib_push(cib_xml)
+
+
+def sspl(cib_xml, push=False):
+    """Create sspl clone resource in pacemaker."""
+    # Using sspl-ll service file according to the content of SSPL repo
+    cmd_sspl = f"pcs -f {cib_xml} resource create sspl-ll systemd:sspl-ll clone op monitor interval=30"
+    SimpleCommand().run_cmd(cmd_sspl)
+
+    if push:
+        cib_push(cib_xml)
+
+
+def io_stack(cib_xml, push=False):
+    """Wrapper to create IO stack related resources."""
+    # TODO: Clarify where shall be sspl in this sequence
+    resources = [ motr_hax, s3auth, haproxy, s3servers, s3bp]
+    for rcs in resources:
+        rcs(cib_xml, push)
+
+def kibana_vip(cib_xml, vip, iface, cidr=24, push=False):
```

Comment:
Rename kibana_vip to mgmt_vip.

Reply:
Renamed everywhere except csm-kibana group which is being discussed in other review finding.

Reply:
Renamed csm-kibana as well.